roles/dnsmasq/templates/dnsmasq-rc.yml

```diff
@@ -1,16 +1,21 @@
 ---
-apiVersion: extensions/v1beta1
-kind: DaemonSet
+apiVersion: v1
+kind: ReplicationController
 metadata:
   name: dnsmasq
   namespace: "{{system_namespace}}"
   labels:
     k8s-app: dnsmasq
+    kubernetes.io/cluster-service: "true"
 spec:
+  replicas: 2
```

Comment:
shall we stick with kubedns replicas related value or autoscaler neglects this count?